src/js/modules/HtmlFromAdmin.js

```diff
@@ -1,18 +1,16 @@
 import React from 'react';
-import { defineMessages, injectIntl } from 'react-intl';
+import PropTypes from 'prop-types';
 
-const i18nMessages = defineMessages({
-  htmlFromAdminSectionTitle: {
-    id: 'HtmlFromAdmin.SectionTitle',
-    defaultMessage: 'What else do I need to know?',
-  }
-});
-
-const HtmlFromAdmin = ({content, intl}) => (
+const HtmlFromAdmin = ({ title, content }) => (
   <div className="coa-HtmlFromAdmin">
-    <h2 className="coa-HtmlFromAdmin__title">{intl.formatMessage(i18nMessages.htmlFromAdminSectionTitle)}</h2>
+    <h2 className="coa-HtmlFromAdmin__title">{title}</h2>
     <div className="coa-HtmlFromAdmin__content" dangerouslySetInnerHTML={{__html: content}} />
   </div>
 );
 
-export default injectIntl(HtmlFromAdmin);
+HtmlFromAdmin.contextTypes = {
+  title: PropTypes.string,
+  content: PropTypes.string.isRequired,
```

Comment:
Not 100% sure why -- but I get a weird warning that says content is required but not set -- tho it's def set.
![screen shot 2018-04-09 at 8 41 11 pm](https://user-images.githubusercontent.com/3039125/38531696-cb344c02-3c36-11e8-8e28-13798da71c63.png)
![screen shot 2018-04-09 at 8 41 43 pm](https://user-images.githubusercontent.com/3039125/38531697-ccce1034-3c36-11e8-9299-b068e580c0cd.png)

Reply:
should've `.propTypes`, not `.contextTypes`